validate_sptx/test_fuzz.py

```diff
@@ -0,0 +1,79 @@
+import os
+from pkg_resources import resource_filename
+
+from .util import SpaceTxValidator
+from .util import Fuzzer
+
+codebook_schema_path = resource_filename("validate_sptx", "schema/codebook/codebook.json")
```

Comment:
@ttung We have a lot of gross path hacks in our codebase that I think this could fix. 

Nice decision here @joshmoore. I didn't know about this!

Reply:
It's completely stolen from elsewhere in the codebase. :smile:

Reply:
... Oh.